.github/workflows/build.yml

```diff
@@ -71,9 +73,6 @@ jobs:
           - remote
         pkg:
           - ./tests
-        include:
-          - pkg: ./...
-            skip-integration-tests: 1
```

Comment:
@jedevc I removed this one as it's already handled in `test-unit` job through `ubuntu-latest` otherise we have dup testing. Let me know if that sounds good.